README.md

```diff
@@ -37,7 +37,6 @@ These libraries contain helper functions, generalized standards, and other tools
 - [Reentrancy](./libs/reentrancy) is used to detect and prevent reentrancy attacks.
 - [Signed Integers](./libs/signed_integers/) is an interface to implement signed integers.
 - [String](./libs/strings/string/) is an interface to implement dynamic length strings that are UTF-8 encoded.
-- [StorageMapVec](./libs/storagemapvec/) is a temporary workaround for a StorageMap<K, StorageVec<V>> type.
```

Comment:
I question the removal of the pointer without removing the library entirely since we must continue to maintain it. At the same time there is a note in a previous PR which states that the library is deprecated so this is probably okay.

Reply:
How we deprecate libraries was to be discussed in a sync.

Reply:
It must have slipped my mind, apologies. This approach is self evident and does not require further explanation. Thanks.